src/core/Table/TablePaginator.tsx

```diff
@@ -89,6 +91,11 @@ export type TablePaginatorProps = {
      * @default (page: number) => `Go to page ${page}`
      */
     goToPageLabel?: (page: number) => string;
+    /**
```

Comment:
Story would be nice because people will miss whether it is possible to hide this label. Also, is it possible to increase story screenshot width that label would be visible there?